src/server/game/DataStores/DB2Structure.h

```diff
@@ -2546,11 +2546,11 @@ struct SpellXSpellVisualEntry
 struct SummonPropertiesEntry
 {
     uint32 ID;
+    uint32 Flags;
     uint32 Category;
     uint32 Faction;
     uint32 Type;
-    int32 Slot;
```

Comment:
Why is there still (after previous fix) a sign change on slot? `TempSummon::InitStats` checks if the value is greater than 0, meaning that someone expected it to be sometimes negative (or 0)